src/elfpy/utils/config.py

```diff
@@ -182,7 +182,8 @@ def get_random_variables(config: Config):
         target_pool_apr=config.simulator.rng.uniform(
             low=config.amm.min_pool_apr, high=config.amm.max_pool_apr
         ),  # starting fixed apr as a decimal
-        fee_percent=config.simulator.rng.uniform(low=config.amm.min_fee, high=config.amm.max_fee),
+        trade_fee_percent=config.simulator.rng.uniform(low=config.amm.min_fee, high=config.amm.max_fee),
+        redemption_fee_percent=config.simulator.rng.uniform(low=config.amm.min_fee, high=config.amm.max_fee),
```

Comment:
Are you sure that these two fees should have a single config parameter defining their range?

Reply:
No I'm not sure.  We can change this if needed.